doc/nrf/links.txt

```diff
@@ -294,9 +295,11 @@
 
 .. _`nRF9160 Hardware Verification Guidelines - UART interface`: https://infocenter.nordicsemi.com/topic/nwp_034/WP/nwp_034/nwp_034_uart_if.html
 
-.. _`Creating an nRF Cloud account`: https://infocenter.nordicsemi.com/topic/ug_nrf91_dk_gsg/UG/common/create_nrfcloud_account.html
+.. _`Creating an nRF Cloud account`:
+.. _`Creating an nRF Connect for Cloud account`: https://infocenter.nordicsemi.com/topic/ug_nrf91_dk_gsg/UG/common/create_nrfcloud_account.html
```

Comment:
Does this mean both versions refer to the same target and they work?

Reply:
Should be. The build's passing. If the links didn't work, the build would break, right?

Reply:
The name change is cosmetic only.  There is only one nRF Cloud.  There is not also some older "nRF Connect for Cloud" living somewhere as well.

Reply:
nRF Connect for Cloud lives in the older releases.

Reply:
@peknis I gather you consider "nRF Connect for Cloud" to be some component in NCS, rather than the cloud service itself?  The cloud service is not something people download, modify, and run for themselves, like "Asset Tracker" in NCS -- it's the service hosted at nrfcloud.com.

Is this the root of our miscommunication?

Reply:
No. It's just that we may have occurrences of nRF Connect for Cloud in the old release notes and we do not touch them. If we keep both versions in the links.txt, the links still work for the old release notes.

Reply:
Got it, thanks.  Sorry for my confusion!